redfish-core/lib/systems.hpp

```diff
@@ -2623,8 +2623,8 @@ inline void handleComputerSystemCollectionGet(
     system["@odata.id"] = "/redfish/v1/Systems/system";
     ifaceArray.emplace_back(std::move(system));
     sdbusplus::asio::getProperty<std::string>(
-        *crow::connections::systemBus, "xyz.openbmc_project.Settings",
-        "/xyz/openbmc_project/network/hypervisor",
+        *crow::connections::systemBus, "xyz.openbmc_project.Network.Hypervisor",
```

Comment:
This part isn't upstream. Why do we need this change?

Reply:
this is a downstream only change. We are enabling hypervisor with IBM flag in the networkd.
So this bmcweb part of code was never changed..